src/sidebar/components/ShareDialog/test/ExportAnnotations-test.js

```diff
@@ -456,6 +466,70 @@ describe('ExportAnnotations', () => {
     });
   });
 
+  context('when copying annotations export to clipboard', () => {
+    [true, false].forEach(exportFormatsEnabled => {
+      it('displays copy button if `export_formats` FF is enabled', () => {
+        fakeStore.isFeatureEnabled.callsFake(
+          ff => exportFormatsEnabled || ff !== 'export_formats',
```

Comment:
I think this would be slightly easier to read as `ff => ff === 'export_formats' ? exportFormatsEnabled : true`